TDP/tdp-44.md

```diff
@@ -0,0 +1,69 @@
+---
+tdp: 44
+title: mStableDAO Restructure
+status: WIP
+author: mZeroNine (@mzero2nine)
+discussions-to: https://forum.mstable.org
+created: 2022-06-20
+---
+
+## Simple Summary
+
+It is proposed to restructure the mStableDAO in order to increase its efficiency as well as overall ability to scale in the future without further re-adjustments down the line in order to prepare for the release of mStable v2. 
+
+Since the restructuring process as a whole is extensive, it is also proposed to split the mStableDAO restructuring into multiple TDPs in order to maintain explicitness in the voting outcome as well as ensuring a digestible format for Meta Governors. The entire process will contain 4 TDPs in total and is planned to conclude by the 3rd week of August 2022 the latest.

```

Comment:
Date in the proposal is not relevant to the proposal. I would not specify this.

Reply:
> So in a nutshell, this TDP is just asking whether we want to restructure or not? Why not propose the how directly in the proposal?

I'm not sure I follow? 

> Date in the proposal is not relevant to the proposal. I would not specify this.
Roger, can change once above comment is sorted :)